src/server/shared.ts

```diff
@@ -46,70 +46,79 @@ export function getAuthUrlOrigin() {
 export function getAuthUrl() {
     const authUrl = process.env.NEXT_PUBLIC_AUTH_URL
     if (!authUrl) {
-        throw new Error("NEXT_PUBLIC_AUTH_URL is not set")
+        throw new Error('NEXT_PUBLIC_AUTH_URL is not set')
     }
     return new URL(authUrl)
 }
 
 export function getRedirectUri() {
     const redirectUri = process.env.PROPELAUTH_REDIRECT_URI
     if (!redirectUri) {
-        throw new Error("PROPELAUTH_REDIRECT_URI is not set")
+        throw new Error('PROPELAUTH_REDIRECT_URI is not set')
     }
     return redirectUri
 }
 
 export function getIntegrationApiKey() {
     const integrationApiKey = process.env.PROPELAUTH_API_KEY
     if (!integrationApiKey) {
-        throw new Error("PROPELAUTH_API_KEY is not set")
+        throw new Error('PROPELAUTH_API_KEY is not set')
     }
     return integrationApiKey
 }
 
 export function getVerifierKey() {
     const verifierKey = process.env.PROPELAUTH_VERIFIER_KEY
     if (!verifierKey) {
-        throw new Error("PROPELAUTH_VERIFIER_KEY is not set")
+        throw new Error('PROPELAUTH_VERIFIER_KEY is not set')
     }
-    return verifierKey.replace(/\\n/g, "\n")
+    return verifierKey.replace(/\\n/g, '\n')
 }
 
-export async function refreshTokenWithAccessAndRefreshToken(refreshToken: string): Promise<RefreshTokenResponse> {
+export async function refreshTokenWithAccessAndRefreshToken(
+    refreshToken: string,
+    activeOrgId?: string
+): Promise<RefreshTokenResponse> {
     const body = {
         refresh_token: refreshToken,
     }
-    const url = `${getAuthUrlOrigin()}/api/backend/v1/refresh_token`
+
+    const queryParams = new URLSearchParams()
+    if (activeOrgId) {
+        queryParams.set('with_active_org_support', 'true')
+        queryParams.set('active_org_id', activeOrgId)
```

Comment:
[nit] the backend only requires the `active_org_id` param for this, but this will also work with `with_active_org_support`, just not necessary